wasmsdk/sdk.go

```diff
@@ -1099,36 +1114,36 @@ func CommitToFabric(this js.Value, p []js.Value) interface{} {
 	return promiseConstructor.New(handler)
 }
 
-func GetAllocationMinLock(this js.Value, p []js.Value) interface{} {
```

Comment:
@Rizary Why this code removed?

Reply:
@cnlangzi has this new feature called `jsbridge` in `wasmsdk/jsbridge`. If you take a look at `wasmsdk/proxy.go` here: https://github.com/0chain/gosdk/pull/301/files#diff-5dca85185f55c99aa4bc09436c42c6721a4c59a9521cd77d067d7d18ae243e15R25 the `GetAllocationMinLock` in `sdk.go` is no longer needed. So we can called directly from `sdk.GetAllocationMinLock`. My commit in this PR is to try jsbridge, yet it's still has a blocker. 

Should we use the normal way first and change it into jsbridge later?

Reply:
Got it, make sense

Reply:
@Rizary , let us remove duplicated code from sdk.go if possible. thanks.